performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -9,8 +9,23 @@ while also allowing you to measure the performance of any piece of code in your
 
 Introduction
 ------------
+The app is centered around image tasks: download, classify, saliency map, and upload. Downloading
+and uploading images is a common developer task. Classifying the image and generating a saliency map
+are tasks that might be more popular now with the success of computer vision algorithms. Classifying
+the image consists of providing categories to which the image belongs, while generating a saliency
+map consists of producing a map that identifies the parts of an image most likely to draw attention.
```

Comment:
Perhaps be more concise with a separate sentence like : "A saliency map identifies the parts of an image most likely to draw attention."

Reply:
Done ✅